Pkgfile

```diff
@@ -299,6 +304,11 @@ vars:
   rhash_sha256: 8e7d1a8ccac0143c8fe9b68ebac67d485df119ea17a613f4038cda52f84ef52a
   rhash_sha512: 00a7e5e058b53ce20ae79509815452ed9cb699d1322b678220b72c61dea3ea2f8fa131acfade8bb6d9f6af913f0c3c472330841181b22314b8755166310c946f
 
+  # renovate: datasource=github-tags extractVersion=^(?<version>.*)$ depName=SELinuxProject/selinux
```

Comment:
we don't need the `extractVersion` though this needs to be added to https://github.com/siderolabs/tools/blob/main/.github/renovate.json#L57

Reply:
@frezbo did I do that right? /m if so